### PR TITLE
add cargo truck class

### PR DESCRIPTION
**Pull request by semakers**

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -11,6 +11,7 @@ public static void main(String[] args) {
         Car car = new Car("Red", "ABCD", 105, 2000, "ABCD", 100, 50);
         Motorcycle motorcycle = new Motorcycle();
         Race race = new Race(vehiclesList, silverstone, 17, 2);
+        CargoTruck cargoTruck = new CargoTruck();
```
Comment (Germonlar) [inline src/main/java/org/example/Main.java]: we said that we're not going to make any modifcation on main.

**Review by Germonlar — CHANGES_REQUESTED** [review]
pls adrian

File: src/main/java/org/example/Main.java (hunk 1 above)
Comment (semakers) [reply src/main/java/org/example/Main.java]: sorry

**Review by Germonlar — APPROVED** [review]
GOOD!

**Pull request merged**